pkg/debug/scripts/collect-info.sh

```diff
@@ -373,7 +373,7 @@ ln -s /persist/netdump      "$DIR/persist-netdump"
 ln -s /persist/kcrashes     "$DIR/persist-kcrashes"
 ln -s /run                  "$DIR/root-run"
 cp -r /sys/fs/cgroup/memory "$DIR/sys-fs-cgroup-memory" >/dev/null 2>&1
-ln -s /persist/SMART_details* "$DIR/"
+stat -t /persist/SMART_details.json > /dev/null 2>&1 && ln -s /persist/SMART_details* "$DIR/"
```

Comment:
Please `if [ -f /persist/SMART_details.json ] etc`, `stat` difficult to read .
Also please squash this commit to the previous one. We don't collect commits iteratively.